docs/griptape-framework/drivers/src/griptape_cloud_ruleset_driver.py

```diff
@@ -0,0 +1,7 @@
+from griptape.drivers import GriptapeCloudRulesetDriver
+from griptape.rules import Ruleset
+
+rulset = Ruleset(
+    name="my_griptape_cloud_ruleset_alias",
```

Comment:
Can you create a real ruleset in the service account that we run integ tests against?